### PR TITLE
Use chat app as testnet bot

### PR DESCRIPTION
**Pull request by bacv**
Initial container for sending ping messages to the testnet every 10 seconds

### CODE REVIEW

File: testnet/scripts/run_nomos_bot.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+echo "I am a container ${HOSTNAME} bot"
+
+exec /usr/bin/nomos-cli disseminate /etc/nomos/libp2p_config.yaml --with-metrics
```
Comment (youngjoon-lee) [inline testnet/scripts/run_nomos_bot.sh:5]: Asking just to understand the script. Do we need to execute this dissemination here before starting the chat bot?

Comment (bacv) [reply testnet/scripts/run_nomos_bot.sh:5]: I knew I've missed something, thanks for catching. Maybe later we'll constantly push file to network, but at the moment this is just a leftover, will remove.

Comment (youngjoon-lee) [reply testnet/scripts/run_nomos_bot.sh]: Aha, I understand, that's why the name of this script is `run_nomos_bot.sh`, not `run_nomos_chatbot.sh`. It makes sense. Thank you!

**Review by danielSanchezQ — APPROVED** [review]
PEAK AI AT WORK

**Pull request merged**